jetty-util/src/main/java/org/eclipse/jetty/util/thread/AutoLock.java

```diff
@@ -22,37 +22,27 @@
 import java.util.concurrent.locks.ReentrantLock;
 
 /**
- * <p>Convenience auto closeable {@link java.util.concurrent.locks.ReentrantLock} wrapper.</p>
- *
+ * Reentrant lock that can be used in a try-with-resources statement.
  * <pre>
- * try (Locker.Lock lock = locker.lock())
+ * try (AutoLock lock = this.lock.lock())
  * {
- *   // something
+ *     // Something
  * }
  * </pre>
  */
-public class Locker
```

Comment:
probably should leave Locker as a deprecated class, just in case others are using it. Remove in 10

Reply:
@gregw this PR is for 10 only.